src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1207,6 +1207,7 @@ class RPCParser
             {"fetch_info", &RPCParser::parseFetchInfo, 0, 1},
             {"gateway_balances", &RPCParser::parseGatewayBalances, 1, -1},
             {"get_counts", &RPCParser::parseGetCounts, 0, 1},
+            {"get_aggregate_price", &RPCParser::parseAsIs, 3, 5},
```

Comment:
This RPC should be added to the `help` message in `Main.cpp`

Reply:
We are not going to offer this command in cli. Will delete.